### PR TITLE
Shen 22.0 Upgrade

### PR DESCRIPTION
**Pull request by rkoeninger**
- [x] New version of kernel, load `init.kl`, call `shen.initialise`.
- [x] Include `feature-expand` and provide feature symbols.
- [x] Use `launcher` extension in `npm run repl`.

### CODE REVIEW

File: scripts/render.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const render = async => {
       Block,
       ...defuns.map(construct),
       Assign(Id('$'), Call(Call(Id('require'), [Literal('../lib/overrides')]), [Id('$')])),
-      ...statements.map(construct));
+      assemble(Statement, construct([s`shen.initialize`])));
```
Comment (tizoc) [inline scripts/render.js]: British spelling!

Comment (rkoeninger) [reply scripts/render.js]: ironic for me to forget

**Comment by rkoeninger** [comment]
@tizoc Example feature list is `[shen-script js node windows]`. Is that reasonable or what is the standard? Should it be `shen/script` instead of `shen-script`?

**Comment by tizoc** [comment]
@rkoeninger sounds good to me. For Shen/Scheme I only added `shen/scheme` for now, will give it more thought in the future (I missed the symbol for the operating system, but will do that in the next release).

`shen-script` is fine btw, I use `shen/scheme` because thats the name of the port, not because the slash has any special meaning.

**Comment by rkoeninger** [comment]
I'm having trouble with the launch extension. In node, `process.argv` is returning:

```
[ 'C:\\Program Files\\nodejs\\node.exe',
  'C:\\Users\\koeningerr\\Workspace\\Me\\ShenScript\\scripts\\repl.js' ]
```

which I convert to a cons list of course. Trying to run with no additional arguments prints:

```
ERROR: Invalid argument: C:\Users\koeningerr\Workspace\Me\ShenScript\scripts\repl.js
Try `C:\Program Files\nodejs\node.exe --help' for more information.
```

`shen.exe` can't run without any arguments now?

**Comment by tizoc** [comment]
@rkoeninger the node.exe part has to be excluded, the first element in the list has to be the program that launches Shen itself.

**Pull request merged**